Makefile

```diff
@@ -0,0 +1,36 @@
+OPAM_PREFIX?=$(DESTDIR)$(shell opam config var prefix)
+OPAM_LIBDIR?=$(DESTDIR)$(shell opam config var lib)
+
+.PHONY: release build install uninstall clean test doc reindent
```

Comment:
+gh-pages

Reply:
That was there by mistake, well spotted. Thanks!